doc/sphinx/language/core/sections.rst

```diff
@@ -65,9 +65,7 @@ usable outside the section as shown in this :ref:`example <section_local_declara
    This can be used to define a term incrementally,
    in particular by relying on the :tacn:`refine` tactic.
    See :ref:`proof-editing-mode`.
-   The proof should be terminated with :cmd:`Defined`. Using :cmd:`Qed` in
-   Let is deprecated. To get the same effect (hiding the body of the
-   proof), add the :attr:`clearbody` attribute to the Let.
+   The proof should be terminated with :cmd:`Defined`.
```

Comment:
IIUC it's also valid to terminate with `Qed`.  Perhaps you could briefly explain that one should used `Defined` unless <_describe when to use `Qed`_>,